CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# dbt_google_ads v0.13.0
+
+## Schema Updates
+
+**2 total changes • 1 possible breaking change**
+| **Model/Column** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| [google_ads__search_term_report](https://fivetran.github.io/dbt_google_ads/#!/model/model.google_ads.google_ads__search_term_report) | New Column |   |  `criterion_id` | **BREAKING:** This may change the model's grain, as a single `keyword_text` can have multiple `criterion_id` values. `criterion_id` is included in uniqueness tests for this model. |
+| [stg_google_ads__search_term_keyword_stats](https://fivetran.github.io/dbt_google_ads_source/#!/model/model.google_ads_source.stg_google_ads__search_term_keyword_stats) | New Column |   |  `criterion_id` |  Parsed out of `keyword_ad_group_criterion` field |
```

Comment:
No need to link to the source docs. We have this already in the transform docs and can keep the user in the same location.
```suggestion
| [stg_google_ads__search_term_keyword_stats](https://fivetran.github.io/dbt_google_ads/#!/model/model.google_ads_source.stg_google_ads__search_term_keyword_stats) | New Column |   |  `criterion_id` |  Parsed out of `keyword_ad_group_criterion` field |
```

Reply:
Adjusting myself directly